src/admin-views/filter_bar/upsell.php

```diff
@@ -6,28 +6,29 @@
  * @since 5.14.0
  * @since TBD Drastically simplified the HTML, changed classes to use new admin styles
  */
+_deprecated_file( __FILE__, 'TBD', 'src/admin-views/settings/upsells/filter_bar.php' );
```

Comment:
We should remove line 2 above, and let the comment be the first thing in the file. Also, we should add `@deprecated` to the file header.